src/ru/geekbrains/lesson7/StringHomework.java

```diff
@@ -0,0 +1,42 @@
+package ru.geekbrains.lesson7;
+
+import java.util.Locale;
+
+public class StringHomework {
+    public static void main(String[] args) {
+
+        String str = "I like Java!!!";
+
+        StringProcess(str);
+    }
+
+    public static void StringProcess(String str) {
```

Comment:
```suggestion
    public static void stringProcess(String str) {
```
методы именуем с прописной буквы